File: app/src/main/java/com/erikzuo/portfolioandroidapp/ui/main/MainViewModel.java
package com.erikzuo.portfolioandroidapp.ui.main;


import android.databinding.ObservableField;
import android.util.Log;

import com.erikzuo.portfolioandroidapp.data.DataManager;
import com.erikzuo.portfolioandroidapp.data.model.MyInfo;
import com.erikzuo.portfolioandroidapp.ui.base.BaseViewModel;
import com.erikzuo.portfolioandroidapp.utils.rx.SchedulerProvider;

import javax.inject.Inject;

/**
 * Created by YifanZuo on 4/2/18.
 */

public class MainViewModel extends BaseViewModel<MainNavigator> {


    @Inject
    public MainViewModel(DataManager dataManager, SchedulerProvider schedulerProvider) {
        super(dataManager, schedulerProvider);


    }

}

File: app/src/main/java/com/erikzuo/portfolioandroidapp/ui/main/MainViewModel.kt
package com.erikzuo.portfolioandroidapp.ui.main


import com.erikzuo.portfolioandroidapp.data.DataManager
import com.erikzuo.portfolioandroidapp.ui.base.BaseViewModel
import com.erikzuo.portfolioandroidapp.utils.rx.SchedulerProvider

import javax.inject.Inject

/**
 * Created by YifanZuo on 4/2/18.
 */

class MainViewModel @Inject
constructor(dataManager: DataManager, schedulerProvider: SchedulerProvider) : BaseViewModel<MainNavigator>(dataManager, schedulerProvider) {

    init {
        loadMyInfo()
    }


}
